From e0a112874d10dbf741c27b107edddedd86d51529 Mon Sep 17 00:00:00 2001
From: Wenzong Fan <wenzong.fan@windriver.com>
Date: Tue, 14 Feb 2017 06:32:35 +0000
Subject: [PATCH] setools4: fixes for cross compiling

* search libsepol from $STAGING_LIBDIR
* fix manual install path as '/usr/share/man/man1'

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Wenzong Fan <wenzong.fan@windriver.com>
Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index ac8a876..df7ebbc 100644
--- a/setup.py
+++ b/setup.py
@@ -78,7 +78,7 @@ class QtHelpCommand(Command):
 
 
 # Library linkage
-lib_dirs = ['.', '/usr/lib64', '/usr/lib', '/usr/local/lib']
+lib_dirs = [os.environ["STAGING_LIBDIR"]]
 include_dirs = []
 
 with suppress(KeyError):
-- 
2.7.4

